scripts/download-import-schema-from-gecko-dev

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+const path = require('path');
+
+const shell = require('shelljs');
+
+// Fail as soon as a shelljs command fails.
+shell.config.fatal = true;
+
+const gitClone = {
+  repo: 'https://github.com/mozilla/gecko-dev',
+  branch: 'beta',
+  paths: [
+    'browser/config/version.txt',
+    'browser/config/version_display.txt',
+    'toolkit/components/extensions/schemas',
+    'browser/components/extensions/schemas',
+    'mobile/android/components/extensions/schemas',
+  ],
+  partialCloneOptions: '--depth 1 --filter=blob:none --no-checkout',
+  tmpDir: path.join('tmp', `gecko-dev--partial-${Date.now()}`),
```

Comment:
Maybe having `tmpDir` as an absolute path would make sense here (on some systems, it might be better).

Also, could we use the OS "tmp dir" feature?

Reply:
sure, I'm not against that, the other script we are currently using is using the ./tmp directory in the addons-linter working tree and so I opted to just stick to that in this alternative script (mainly because of "weekend lazyness" :-P).

Separately I'll need to double-check if in a github workflow I may need to point TMPDIR elsewhere, but that isn't going to be an issue.